openbsd/OpenBSDProcessList.c

```diff
@@ -176,30 +173,25 @@ static double getpcpu(const struct kinfo_proc* kp) {
    if (fscale == 0)
```

Comment:
Float equal comparison, use `fscale < 1E-6` instead.

Reply:
`fscale` is of type `long`